tqecjs/src/workspace/QubitClass.js

```diff
@@ -0,0 +1,130 @@
+import { Graphics, Text } from 'pixi.js';
+
+/**
+ * Qubit class
+ * @extends Graphics
+ * @constructor
+ * @param {number} x - The x position of the qubit
+ * @param {number} y - The y position of the qubit
+ */
+export default class Qubit extends Graphics {
+	constructor(x, y, radius = 5, color = 0x000000, gridSize = 50) {
+		super();
+		// UI properties
+		this.eventMode = 'static';
+		this.buttonMode = true;
+		this.cursor = 'pointer';
+		this.globalX = x;
+		this.globalY = y;
+		this._createCircle(x, y, radius, color);
+		this.maxNeighborDist = 2 * this.gridSize;
+		this.neighbors = [];
+		this.gridSize = gridSize;
+		// Adjacent (degree 1) qubits
+		// QC properties
+		this.quantumState = '0'; // NOTE: In javascript, becareful updating values such as "this", "state", etc.
```

Comment:
Can `quantumState` be anything other than 0 or 1? My intuition is yes, but I'm unsure of the representation here

Reply:
unused at the moment. it's just a dummy variable

Reply:
removed